core/primitives/src/runtime/config.rs

```diff
@@ -55,6 +55,7 @@ impl RuntimeConfig {
 /// but i) may have it’s `max_gas_burnt_view` limit adjusted and ii) provides
 /// a method which returns configuration with adjustments done through protocol
 /// version upgrades.
+/// TODO #4649: deprecated after RuntimeConfigStore creation, remove it
```

Comment:
One trick to refactoring large code-bases is to mark deprecated functionality as `#[deprecated]`, and mark all call-sites as `#[allow(deprecated)]`. You can do this in a mechanical fashion, and it makes sure that people won't introduce new usages. 

You can then fix call-sites one by one in the follow up PRs. 

NOt saying that we should do this in this PR, it's just a general observation that incomplete refactors often lead to code debt, and that's something to keep an eye (or tools) on!